02/app.js

```diff
@@ -17,9 +21,8 @@ let n = 4;
 let aNum = parseInt(prompt("Podaj liczbę a:"));
 let nNum = parseInt(prompt("Podaj liczbę n:"));
 
-
-while (aNum !== a && nNum !== n) {
+while (aNum !== a || nNum !== n) {
   aNum = parseInt(prompt("Podaj liczbę a:"));
   nNum = parseInt(prompt("Podaj liczbę n:"));
 }
-console.log(Math.pow(a, n));
\ No newline at end of file
+console.log(`${a} x ${a} x ${a} x ${a} = ${Math.pow(aNum,nNum)}`);
```

Comment:
Ten zapis nie będzie poprawny zawsze (tylko gdy n=4) - trzeba zrobić to inaczej.
Podpowiedź: potrzebujesz dodatkowej zmiennej do której będziesz w pętli zapisywać częściowy wynik tj. 
```str += `${a} x` ```